README.md

```diff
@@ -28,7 +28,7 @@ It contains the following projects:
 Building
 --------
 
-### Third-party
+### Third-party (Windows Only)
```

Comment:
That's actually not true: the third-parties may be used on POSIX systems as well, when the package manager doesn't provide the appropriate versions.

That being said, I never compile the third-parties on Linux but do so on OSX. Perhaps this could be reworded to be more explicit ?